physionet-django/events/templates/events/event_entries.html

```diff
@@ -6,6 +6,7 @@
           <th>Full name</th>
           <th>Email</th>
           <th>Credentialed</th>
+          <th>CoHost</th>
```

Comment:
H is not capitalised.

Reply:
Thanks @tompollard , I made a new commit and squashed the change